src/custom/pages/Profile/index.tsx

```diff
@@ -210,6 +211,9 @@ export default function Profile() {
               <ExtLink href={getBlockExplorerUrl(chainId || 1, V_COW_CONTRACT_ADDRESS[chainId || 1], 'address')}>
                 Contract ↗
               </ExtLink>
+              <CopyHelper toCopy={V_COW_CONTRACT_ADDRESS[chainId || 1]}>
```

Comment:
don't use "1" use SupportedChainId.MAINNET pls

Reply:
Updated all instances of it.